#include "ChunkMasterServiceImpl.hpp"
int ChunkMasterServiceImpl::AllocatedUploadSlices(const ::chunkmaster::AllocateUploadSlicesReq & oReq, ::chunkmaster::AllocateUploadSlicesRsp & oResp) {
    return -1;
}
int ChunkMasterServiceImpl::FinishUploadSlice(const ::chunkmaster::FinishUploadSliceReq & oReq, ::chunkmaster::FinishUploadSliceRsp & oResp) {
    return -1;
}
int ChunkMasterServiceImpl::ReportChunkInformation(const ::chunkmaster::ReportChunkInformationReq & oReq, ::chunkmaster::ReportChunkInformationRsp & oResp) {
    return -1;
}
